packages/amplication-client/src/Topics/TopicList.tsx

```diff
@@ -39,19 +39,18 @@ export const TopicList = React.memo(
     );
     const history = useHistory();
 
-    const { data, loading, error } = useQuery<TData>(GET_ROLES, {
+    const { data, loading, error } = useQuery<TData>(GET_TOPICS, {
       variables: {
-        id: resourceId,
+        where: { 
+          resource: { id: resourceId }, 
+          displayName: {
+            contains: searchPhrase,
+            mode: models.QueryMode.Insensitive,
```

Comment:
Did you test what happens when searchPhrase is an empty string?

Reply:
Reverted change.